single_kernel_mongo/config/statuses.py

```diff
@@ -379,71 +379,104 @@ def refreshing_needs_resume(
 class LdapStatuses(Enum):
     """Ldap Statuses."""
 
-    INVALID_LDAP_USER_MAPPING = StatusObject(
-        status="blocked",
-        message="Invalid LdapUserToDnMapping, please update your config.",
+    ACTIVE_IDLE = StatusObject(
+        status="active",
+        message="",
     )
-    INVALID_LDAP_QUERY_TEMPLATE = StatusObject(
-        status="blocked",
-        message="Invalid LDAP Query template, please update your config",
+    WAITING_FOR_DATA_AND_CERTS = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP data and certificates.",
+        short_message="LDAP is not ready...",
     )
-    CONFIGURING_LDAP = StatusObject(
-        status="maintenance", message="Configuring LDAP", running="blocking"
+    WAITING_FOR_CERTS = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP certificates...",
     )
-    INVALID_LDAP_REL_ON_SHARD = StatusObject(
-        status="blocked",
-        message="Cannot integrate LDAP with shard.",
+    WAITING_FOR_DATA = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP data...",
     )
-    TLS_REQUIRED = StatusObject(
+    INVALID_USER_MAPPING = StatusObject(
         status="blocked",
-        message="TLS is mandatory for LDAP transport.",
+        message="Invalid LDAP to DN mapping config.",
+        action="Check the logs and verify the configuration in the ldap relation.",
+        check="LDAP configuration validation failed.",
     )
-    LDAP_REQUIRED = StatusObject(
+    INVALID_QUERY_TEMPLATE = StatusObject(
         status="blocked",
-        message="GLauth TLS is integrated but LDAP is not.",
+        message="Invalid LDAP query template config.",
+        action="Check the logs and verify the configuration in the ldap relation.",
+        check="LDAP configuration validation failed.",
     )
-    LDAP_SERVERS_MISMATCH = StatusObject(
+    INVALID_LDAP_REL_ON_SHARD = StatusObject(
         status="blocked",
-        message="mongos and config-server not integrated with the same ldap server.",
+        message="The ldap relation cannot be used by shards.",
+        short_message="Invalid ldap relation.",
+        action="Remove the ldap relation (ldap interface) from this application.",
+        check="Relation validation failed.",
     )
-    WAITING_FOR_DATA = StatusObject(
-        status="waiting",
-        message="Waiting for both LDAP data and Glauth certificates.",
+    MISSING_CERT_TRANSFERT_REL = StatusObject(
+        status="blocked",
+        message="Missing ldap-certificate-transfer relation.",
+        short_message="TLS is mandatory for LDAP transport.",
+        action="Add the ldap-certificate-transfer relation between LDAP application and this application.",
```

Comment:
On VM, Glauth is deployed on k8s and then exposed as an offer to the VM model. I would reflect that in the action message maybe?
```suggestion
        action="Add the ldap-certificate-transfer relation between LDAP application/offer and this application.",
```